counter/models.py

```diff
@@ -993,7 +996,7 @@ class CashRegisterSummaryItem(models.Model):
     )
     value = CurrencyField(_("value"))
     quantity = models.IntegerField(_("quantity"), default=0)
-    check = models.BooleanField(_("check"), default=False)
```

Comment:
Ce champ représente si cet élément est un chèque, un chèque de banque, le truc pour payer. Rien à voir donc avec le check d'une checkbox, et ce n'est surtout pas un verbe. Donc le renommage pourrait être `is_check`, mais pas `is_checked`, parce que ça n'a plus aucun sense. Je suggère de ne simplement pas renommer, mais possiblement ajouter un commentaire expliquant ce qu'est un chèque de banque (j'avoue que je l'avais pas vu arrivé, celle là, en 2015 XD)

Reply:
EDIT: j'avais pas vu que le renommage était lié à un built in de Django. `is_check` est donc bien mieux que `is_checked`, et ça tient toujours pour le commentaire.

Reply:
Corrigé